source/common/protobuf/utility.cc

```diff
@@ -31,7 +31,11 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
 }
 
 Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& message) {
-  Protobuf::util::JsonOptions json_options;
+  Protobuf::util::JsonPrintOptions json_options;
+  // By default, proto field names are converted to camelCase when the message is converted to JSON.
+  // Setting this option makes debugging easier because it keeps field names consistent in JSON
+  // printouts.
+  json_options.preserve_proto_field_names = true;
```

Comment:
I may be missing something here, but isn't `Protobuf::util::MessageToJsonString` defined by the external protobuf lib?  But since it looks like we'll be using this a lot with the same options, I think it makes sense to wrap it in a function inside `MessageUtil`.